Examples/UIExplorer/js/FlatListExample.js

```diff
@@ -124,6 +130,16 @@ class FlatListExample extends React.PureComponent {
           renderItem={this._renderItemComponent}
           shouldItemUpdate={this._shouldItemUpdate}
           viewabilityConfig={VIEWABILITY_CONFIG}
+          onScroll={
+            Animated.event([{
+              nativeEvent: { contentOffset: { y: this.state.scrollY } }
+            }], {
+              useNativeDriver: true,
+              listener: (e) => {
+                this._listRef.onScroll(e);
```

Comment:
This is too fragile I think, unless we can come up with a robust way to detect if it's left out?